docs/pages/versions/unversioned/guides/setup-native-firebase.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Using the native Firebase SDK
+sidebar_title: Native Firebase
+---
+
+It's possible to use Firebase through the web SDK, built only in JavaScript, or the native SDK, which is built in native code for iOS and Android. The web SDK only provides access to some Firebase features and the most notable limitation is the lack of support for Firebase Analytics. If this is important for you, it may make sense to install the native SDK in your app. Using the bare workflow it is fairly straightforward to install the native Firebase SDK, but it's a bit trickier in the managed workflow currently.
```

Comment:
Doc mentions that Firebase Analytics is not supported in the JS SDK on react-native. It does seem to exist in the JavaScript SDK. Can you be more specific here, or put a reference to a source/article why it is not supported?